packages/components/button/src/Button.tsx

```diff
@@ -46,3 +46,5 @@ export const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
     )
   }
 )
+
+Button.displayName = 'Button'
```

Comment:
If we decide to do that... it must be applied in all packages n the templates

Reply:
Yes